gateway/gateway-requests/src/types.rs

```diff
@@ -126,6 +132,21 @@ impl ClientControlRequest {
             iv: iv.to_base58_string(),
         }
     }
+
+    pub fn new_enc_bandwidth_credential(credential: Credential, shared_key: &SharedKeys) -> Self {
+        ClientControlRequest::BandwidthCredential {
+            enc_credential: shared_key
+                .encrypt_and_tag(&bincode::serialize(&credential).unwrap(), None),
```

Comment:
I'm not 100% sure if a zero IV is fine here because if you send the same credential again, the ciphertext will be identical. We should ask @aniampio about her opinion. But my gut feeling is as follows: we should either 
a) always explicitly randomise the credential before sending it to ensure the bytes representation would be different each time even for the same underlying cryptographic claims
b) use non-zero IV

Reply:
Switched to using a fresh IV each time a BandwidthCredential message is created.